composer.json

```diff
@@ -48,7 +48,11 @@
         "doctrine/orm": "The Doctrine ORM integration is optional in the bundle."
     },
     "autoload": {
-        "psr-4": { "Doctrine\\Bundle\\DoctrineBundle\\": "" }
+        "psr-4": { "Doctrine\\Bundle\\DoctrineBundle\\": "" },
+        "exclude-from-classmap": ["Tests/"]
```

Comment:
in this case we will need explicitly provide 5 folders and 4 files in root directory